frontend/src/main/components/SectionSearch/SectionSearchForm.js

```diff
@@ -0,0 +1,94 @@
+import { useState, useEffect } from "react";
+import { Form, Button, Container, Row, Col } from "react-bootstrap";
+
+import { allTheLevels } from "fixtures/levelsFixtures";
+import { quarterRange } from "main/utils/quarterUtilities";
+
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+import SingleSubjectDropdown from "../Subjects/SingleSubjectDropdown";
+import SingleLevelDropdown from "../Levels/SingleLevelDropdown";
+import { useBackendMutation } from "main/utils/useBackend";
+
+const SectionSearchForm = ({ fetchJSON }) => {
+  const quarters = quarterRange("20084", "20222");
+
+  // Stryker disable all : not sure how to test/mock local storage
```

Comment:
This needs to be removed as we are not mocking local storage here, and it turns off mutation coverage for the entire rest of the file.

If you need it for these three lines, it need a matching 

// Stryker enable all 

after the affected lines.

Otherwise, this is "cheating", not in the sense of academic dishonesty, but in the sense of "getting around the requirement to pass mutation testing".